src/06_domains/cittadini-app/99_locals.tf

```
@@ -40,4 +40,8 @@ locals {
   monitor_resource_group_name     = "${local.project_core}-monitor-rg"
   monitor_action_group_slack_name = "SlackPagoPA"
   monitor_action_group_email_name = "PagoPA"
+
+  azurerm_log_analytics_workspace_log_analytics_name = "${local.project_core}-law"
+  azurerm_log_analytics_workspace_log_analytics_resource_group_name = "${local.project_core}-monitor-rg"
```

let's keep simpler these names
```suggestion
  log_analytics_name = "${local.project_core}-law"
  log_analytics_resource_group_name = "${local.project_core}-monitor-rg"
```

---

moreover, the monitor resource group is already referenced:
see data.azurerm_resource_group.monitor_rg"